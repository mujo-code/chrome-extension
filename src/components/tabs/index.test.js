import React from 'react'
import { create } from 'react-test-renderer'
import { Tabs } from '.'

jest.mock('../../hooks/use-extension')
// eslint-disable-next-line import-order-alphabetical/order
const { useExtension } = require('../../hooks/use-extension')

test('Tabs component matches snapshot', () => {
  useExtension.mockReturnValue({ tabs: ['foo', 'bar'] })
  const tree = create(<Tabs />).toJSON()
  expect(tree).toMatchSnapshot()
})
