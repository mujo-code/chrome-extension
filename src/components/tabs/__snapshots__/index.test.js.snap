// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs component matches snapshot 1`] = `
<div
  className="css-1xhj18k"
>
  <div
    className="css-17xbw2"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    overflow="hidden"
    position="relative"
  >
    <div
      backgroundColor="saltBox"
      className="css-u4k9dx"
      height="m"
      position="absolute"
      width="m"
    />
    <p
      className="css-1hhdzfd"
      color="color"
    >
      foo
    </p>
  </div>
  <div
    className="css-17xbw2"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    overflow="hidden"
    position="relative"
  >
    <div
      backgroundColor="saltBox"
      className="css-u4k9dx"
      height="m"
      position="absolute"
      width="m"
    />
    <p
      className="css-1hhdzfd"
      color="color"
    >
      bar
    </p>
  </div>
</div>
`;
